methods/array_ops/src/pg_gp/array_ops.c

```diff
@@ -702,8 +702,14 @@ array_sub(PG_FUNCTION_ARGS){
 PG_FUNCTION_INFO_V1(array_mult);
 Datum
 array_mult(PG_FUNCTION_ARGS){
-    if (PG_ARGISNULL(0)) { PG_RETURN_NULL(); }
-    if (PG_ARGISNULL(1)) { PG_RETURN_NULL(); }
+    if (PG_ARGISNULL(0) && PG_ARGISNULL(1)) { PG_RETURN_NULL(); }
```

Comment:
Agree with @decibel . Since array_mult/prod is not being used in SVM, we can remove the change from here and have a separate discussion for it.